src/Engine/Systems/Subsystems/PhysicsRenderBridge.hpp

```
@@ -0,0 +1,46 @@
+		m_frameBuffers.publish();
+
+		m_staleBuffer.store(false, std::memory_order_release);
+	}
+
+
+	/* Returns a copy of the latest snapshot.
+	   @param outFrame: The frame to be populated with the latest snapshot.
+	   
+	   @return False if no new frame has been published since last consume (in which case outFrame is populated with the most recent old snapshot), otherwise True.
+	*/
+	inline bool consume(Buffer::PhysRendFramePacket &outFrame) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)